flycheck-rust.el

```diff
@@ -94,6 +95,32 @@ Return non-nil if PROJECT-ROOT is a binary crate, nil otherwise."
   (let ((root-dir (file-name-directory project-root)))
     (file-exists-p (expand-file-name "src/main.rs" root-dir))))
 
+(defun flycheck-rust-find-target (file-name)
+  "Find and return the cargo target associated with the given file.
+
+FILE-NAME is the name of the file that is matched against the
+'src_path' value in the list 'targets' returned by 'cargo
```

Comment:
I wasn't sure what to use because those weren't elisp value I was quoting.  Good to know.